app/create.js

```diff
@@ -17,74 +17,113 @@ define(['app/config', 'app/database', 'jquery', 'bootstrap', 'parsley', 'app/sel
 			numberOfFormItems--;
 		}
 	}
-  
-	// Function for adding elements to the form
-	function add_element() {
-		var newElement = $('#item-layout').clone(true).appendTo("#items table").removeAttr("id");
-		newElement.html(giveId(newElement.html(), formItemId));
-    newElement.html(giveRequired(newElement.html()));
+
+	function add_element_to_form() {
```

Comment:
The comments already show what the functions called by this method should do, and what their name should be, only thing left to do is introducing the functions.